src/frames.ts

```diff
@@ -730,6 +730,26 @@ export class Frame {
         (handle, deadline) => handle.focus());
   }
 
+  async textContent(selector: string, options: types.TimeoutOptions = {}): Promise<string> {
+    return await this._retryWithSelectorIfNotConnected('textContent', selector, options,
+        (handle, deadline) => handle.textContent() as Promise<string>);
```

Comment:
Hmm... We should have the same return type here and in ElementHandle.textContent.

Reply:
Done.